ballerina/engine_utils.bal

```diff
@@ -85,12 +85,8 @@ isolated function createSchema(string schemaString) returns readonly & __Schema|
     'class: "io.ballerina.stdlib.graphql.runtime.engine.Engine"
 } external;
 
-isolated function executeSubscription(ExecutorVisitor visitor, parser:FieldNode fieldNode, any result) = @java:Method {
-    'class: "io.ballerina.stdlib.graphql.runtime.engine.Engine"
-} external;
-
-isolated function getSubscriptionResult(ExecutorVisitor visitor,
-                                        parser:FieldNode node) returns any|error = @java:Method {
+isolated function executeSubscriptionResource(Context context, service object {} serviceObject, parser:FieldNode node)
+    returns any|error = @java:Method {
     'class: "io.ballerina.stdlib.graphql.runtime.engine.Engine"
 } external;
```

Comment:
Shall we move this to the `engine.bal`, since the other similar functions are defined there?